presenters/trial.js

```diff
@@ -58,6 +58,17 @@ function decorateDocuments(documents) {
   return documentsByCategory;
 }
 
+function decorateFDADocuments(documents) {
+  const docs = _.filter(documents, (doc) => doc.source_id === 'fda');
+  _.map(docs, (doc) => {
+    if (doc.fda_approval) {
+      // eslint-disable-next-line no-param-reassign
+      doc.application_id = doc.fda_approval.fda_application.id;
+    }
+  });
```

Comment:
Well, I can't think of any javascriptish way to change the original value while iterating, other than working on a copy of the original array.